src/Microsoft.AspNet.Mvc.WebApiCompatShim/WebApiCompatShimServiceCollectionExtensions.cs

```diff
@@ -15,6 +17,10 @@ public static IServiceCollection AddWebApiConventions(this IServiceCollection se
             // The constructors on DefaultContentNegotiator aren't DI friendly, so just
             // new it up.
             services.AddInstance<IContentNegotiator>(new DefaultContentNegotiator());
+            services.Configure<MvcOptions>(options =>
+            {
+                options.ValidationExcludeFilters.Add(typeof(HttpRequestMessage));
```

Comment:
Because Webapi binds HttpRequestMessage if it is an action parameter automatically, we do not want to look into its properties.